10_code/50_analysis.py

```diff
@@ -0,0 +1,158 @@
+# %%
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from plotnine import *
+import os
+
+# %%
+# Read data
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/main/20_intermediate_files/pop_mortality_merged_no_imputation.csv"
+)
+
+# %%
+# Take a peep:
+df.head()
```

Comment:
I think this can be removed now that the file has been converted to .py

Reply:
thanks for the reminder. got that removed